open-media-match/Dockerfile

```diff
@@ -1,12 +1,15 @@
 FROM python:3.11-bullseye
```

Comment:
One issue you'll run into is that the test CI really wants to try and run the tests in this docker, which will fail if it has a CMD. 

The workaround is to just swap the CI over, doing something like I did in https://github.com/facebook/ThreatExchange/pull/1375 - I think it's fine for the test CI to be disconnected from this docker file, but I dunno

Reply:
So we probably want a test container for CI? I can take a pass.

Reply:
I looked into the workflow a bit more. Looks like it's working fine since the workflow is installing its own dependencies in the container. Locally it succeeds for me as well.

I do think it's better to create a test container, but for now this will do?